srcpkgs/python3-GitPython/template

```diff
@@ -30,6 +31,13 @@ pre_check() {
 	yes | ./init-tests-after-clone.sh
 	export GIT_PYTHON_TEST_GIT_REPO_BASE="${PWD}"
 
-	git config --global user.email "void@example.com"
-	git config --global user.name "void"
+	git config user.email "void@example.com"
+	git config user.name "void"
```

Comment:
i think these need to be --global

Reply:
I don't think it makes a difference, but for the sake of reducing the diff size I've made them global again.

Reply:
I was seeing test failures in ci without the global

Reply:
ah, I didn't look at those apparently :see_no_evil: 

And it probably was still working for me locally because the previous runs had polluted the masterdir

Reply:
anyway, this is good now then, I'd assume?